templates/poster.html

```diff
@@ -1,4 +1,22 @@
 {% extends "base.html" %}
+
+{% block head %}
+
+{{ super() }}
+<meta name="citation_title" content="{{paper.content.title}}" />
+{% for author in paper.content.authors %}
+<meta name="citation_author" content="{{author}}" />
+{% endfor %}
+<meta name="citation_publication_date" content="{{config.data}}" />
+<meta name="citation_conference_title" content="{{config.name}}" />
```

Comment:
You may consider to add a new field in config, e.g., proceeding_booktitle

Reply:
Sure, would this also be for key `config.name`?

Thanks for your comments!

Reply:
Sorry I didn't quite follow your question ... what would it look like?

Reply:
I misunderstood this first time round - have made change as suggested now